surveys/2024-04/questions.md

```diff
@@ -0,0 +1,142 @@
+    - x64
+    - ia32
+    - arm
+    - ppc
+    - s390
+    - Other ?
+
+- How do you get your Node.js executables? (multiple choice)
+    - With a Node.js version manager: nvm, n, nave, nvs, volta, etc.
+    - Using the official installer
+    - Using package managers like npm or yarn and install Node.js as a package
+    - Using a system package manager: apt-get, brew, dnf, yum …
+    - Downloading binaries directly from Nodejs.org
+    - Building Node.js from Source
+    - Other
+
+- What package manager do you use ? (multiple choice - Other should give an input)
+    - npm
+    - yarn
+    - cnpm
+    - pnpm
+    - Other ? 
+
+- If you use a version manager which one do you use (multiple choice - Other should give an input)
+    - nvm
+    - n
+    - nave
+    - nvs
+    - volta
+    - Other ?
+
+## Project Priorities and Direction
+
+- Are you aware of the defined Node.js [Technical Priorities](https://github.com/nodejs/node/blob/main/doc/contributing/technical-priorities.md)? (single choice)
+    - Yes
+    - No
+
+- Which of the current technical priorities are important to you ? (multiple choice)
+    - Modern HTTP
+    - Suitable types for end-users
+    - Documentation
+    - WebAssembly
+    - ESM
+    - Support for features from the latest ECMAScript spec
+    - Observability
+    - Permissions/policies/security model
+    - Better multithreaded support
+    - Single Executable Applications
+
+- Are there technical priorities that you believe are missing (open question)
+
+- What is important to you ? (multiple choice)
+    - Good understanding of the direction of the project
+    - Ability to affect the direction of the project
+    - Consumable APIs and docs
+    - Predictable and stable releases
+    - Innovation at a consumable pace
+    - Easy Installation
+    - Easy issue reporting, resolution and collaboration
+    - Broad deployment platform support
+    - Broad desktop platform support
+    - Consistent and intuitive error handling
+    - Runtime diagnostic tooling
+    - Development time diagnostic tooling
+    - Relevant APIs in core
+    - Module/dependency info and management
+    - Ways to fund their work
+    - Ability to assess impact of changes they make
+    - Reasonable resource usage/performance
+    - Good security and CVE practices in the project
+    - Better CVE management in the ecosystem
+    - Good CI infrastructure and experience in the project
+    - Supportive Collaborators and Environment in the project
+    - Better ways to build consensus in the project
+    - Easy contribution workflow
+    - Ability to embed and bundle the Node.js runtime
+    - A well maintained and secure standard library
+    - Assets that show Node.js is a good choice
+
+- What is important to you that is not in this list? (open question)
+
+## Technical Questions
+
+- Are you using native ESM (e.g. `"type": "module"`,  `.mjs` or other solutions) in your Node.js Development?
```

Comment:
I would add, which node.js version are you using in production 😅

Reply:
This may not be clear enough to users who are doing faux ESM (that is, they are actually running CJS). Perhaps it's worth emphasizing "without using a transpiler such as Babel or TypeScript"

Reply:
You can use native esm with ts.

Reply:
```suggestion
- Regardless of how your code is written in its original form, when it runs in production, does it contain ES module syntax (`import`/`export`)?
```